lib/Doctrine/Common/Cache/CacheProvider.php

```diff
@@ -201,6 +216,25 @@ private function getNamespaceVersion() : int
         return $this->namespaceVersion;
     }
 
+    /**
+     * Return a map containing the passed ids as keys and
+     * their namespaced versions as keys
+     *
+     * @param array $ids The ids that will be namespaced
```

Comment:
We should be more explicit about the data in the array, maybe `string[]` is more accurate here?